internal/deepdoc/parser/pdf/inference/client.go

```
@@ -225,14 +225,25 @@ func (c *Client) OCRRecognize(ctx context.Context, cropped image.Image) ([]pdf.O
-	resp, err := c.httpClient.Get(c.baseURL + "/health")
+	ctx, cancel := context.WithTimeout(context.Background(), healthProbeTimeout)
+	defer cancel()
+	req, err := http.NewRequestWithContext(ctx, http.MethodGet, c.baseURL+"/health", nil)
+	if err != nil {
+		return false
+	}
+	resp, err := c.httpClient.Do(req)
 	if err != nil {
 		return false
 	}
 	resp.Body.Close()
```

golangci-lint (2.12.2)

[error] Error return value of `resp.Body.Close` is not checked

(errcheck)